### PR TITLE
Updates for 1.11.3

### PR DESCRIPTION
**Pull request by mizzioisf**
Updated on 3rd March 2022

### CODE REVIEW

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

**Comment by dbmalkovsky** [comment]
Doesn't the User Guide need to have new information based on the changes in OP-759?

**Comment by mwithi** [comment]
> Doesn't the User Guide need to have new information based on the changes in [OP-759](https://openhospital.atlassian.net/browse/OP-759)?

not necessarly, every issue that needs documentation should have it's own PR, which means the one for OP-759 was still missing, thanks!

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -296,43 +296,53 @@ Open a terminal and navigate to the _sql/_ folder (e.g.):
...
+C:\Users\OH\OpenHospital-v1.11.3\sql
 
 ----
 
 and run the following command to connect to the database server (e.g.):
 
 [source,shell]
 ----
-C:\Users\OH\OpenHospital-v1.11.2\sql> mysql -u root -p [-h hostname/ip address]
+C:\Users\OH\OpenHospital-v1.11.3\sql> mysql -u root -p [-h hostname/ip address]
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc:309]: mysql here? and in line 313?  At least line 313 should say "MariaDB client", no?

Comment (mizzioisf) [reply doc_admin/AdminManual.adoc:309]: No David, MariaDB client is called...mysql :-)

Comment (mizzioisf) [reply doc_admin/AdminManual.adoc:309]: I'm sorry, you are definitely right for row 313 !

Comment (mizzioisf) [reply doc_admin/AdminManual.adoc:309]: Fixed, thank you !

**Review by mwithi — APPROVED** [review]
LGTM!

**Pull request merged**